.buildkite/pipelines/lucene-snapshot/update-branch.yml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - label: Update lucene-snapshot branch
+    command: .buildkite/scripts/lucene-snapshot/update-branch.sh
+    timeout_in_minutes: 15
+  - wait
+  - trigger: "elasticsearch-lucene-snapshot-tests"
```

Comment:
Shouldn't we run the tests with the new snapshot and _then_ commit/push the change?

Reply:
Well, I suppose that's a question for @rjernst. But given that the purpose of this branch is to have a space to work on making sure Elasticsearch has support for upcoming releases of lucene, it's probably better to just keep it up-to-date all the time, even if it's failing. That way, people don't have to deal with any snapshot handling when they're working locally, they just `git pull`.